docs/start.md

```diff
@@ -0,0 +1,207 @@
+to pull the API and Stardog Docker images
+and then launch both via `docker compose` (Option 1).
+
+Your license file has to be in the `STARDOG_HOME` directory.
+
+## Setup for the first run
+
+When you launch the Stardog graph for the first time,
+there are a couple of setup steps that need to be done. 
+These will not have to be repeated for subsequent starts.
+
+To intereact with the Stardog graph, 
+you have two general options:
+
+1. Send HTTP request against the HTTP API of the Stardog graph instance (e.g. with `curl`). See [https://stardog-union.github.io/http-docs/](https://stardog-union.github.io/http-docs/) for a full reference of API endpoints
+2. Use the free Stardog-Studio web app. See the [Stardog documention](https://docs.stardog.com/stardog-applications/dockerized_access#stardog-studio) for instruction to deploy Stardog-Studio as a Docker container.
+
+
+!!! info 
+    Stardog-Studio is the most accessible way 
+    of manually interacting with a Stardog instance. 
+    Here we will focus instead on using the HTTP API for configuration,
+    as this allows programmatic access.
+    All of these steps can also be achieved via Stardog-Studio manually.
+    Please refer to the 
+    [official docs](https://docs.stardog.com/stardog-applications/studio/) to learn how.
+
+
+### Change the superuser password
+
+When you first launch Stardog, 
+a default `admin` user with superuser privilege
+will automatically be created for you.
+You should first change the password of this user:
+
+
+```console
+curl -X PUT -i -u "admin:admin" http://localhost:5820/admin/users/admin/pwd \
+--data '{"password": "NewPassword"}'
+```
+
+### Create a new user
+
+The `.env` file created as part of the `docker compose` setup instructions
+declares the `USERNAME` and `PASSWORD` for the API user.
+The API will send requests to the graph using these credentials.
+When you launch Stardog for the first time, 
+we have to create this user:
+
+```console
+curl -X POST -i -u "admin:NewPassword" http://localhost:5820/admin/users \
+-H 'Content-Type: application/json' \
+--data '{
+    "username": "NewUser",
+    "password": [
+        "NewUserPassword"
+    ]
+}'
+```
+
+Confirm that the new user exists:
+
+```console
+curl -u "admin:NewPassword" http://localhost:5820/admin/users
+```
+
+!!! note
+    Make sure to use the exact `USERNAME` and `PASSWORD` you
+    defined in the `.env` file when creating the new user.
+    Otherwise the API will not have the correct permission
+    to query the graph.
+
+### Create new database
+
+When you first launch Stardog,
+there are no graph databases.
+You have to create a new one to store
+your metadata.
+
+If you have defined a custom `GRAPH_DB` name in the `.env` file,
+make sure to create a database with a matching name.
+By default the API will query a graph database
+with a name of `test_data`.
+
+```console
+curl -X POST -i -u "admin:admin" http://localhost:5820/admin/databases \
```

Comment:
Since the password for `admin` is changed and we use the `NewPassword` in other instruction, I believe this should be `admin:NewPassword`